cli/script/command-executor.ts

```diff
@@ -1267,20 +1272,19 @@ export const release = (command: cli.IReleaseCommand): Promise<void> => {
     lastTotalProgress = currentProgress;
   };
 
-  const updateMetadata: PackageInfo = {
+  const updateMetadata: ReleasePackageInfo = {
     description: command.description,
     isDisabled: command.disabled,
     isMandatory: command.mandatory,
     rollout: command.rollout,
+    isBundlePatchingEnabled: command.isPatch ?? false,
   };
 
-  console.log('updateMetaData ::', updateMetadata);
 
   return sdk
     .isAuthenticated(true)
     .then((isAuth: boolean): Promise<void> => {
-      console.log('authenticated making release call');
-      return sdk.release(command.appName, command.deploymentName, filePath, command.appStoreVersion, updateMetadata, uploadProgress);
+      return sdk.release(command.appName, command.deploymentName, filePath, command.appStoreVersion, updateMetadata, uploadProgress, command.compression ?? 'deflate');
```

Comment:
```suggestion
      return sdk.release(command.appName, command.deploymentName, filePath, command.appStoreVersion, updateMetadata, uploadProgress, command.compression ?? DEFAULT_COMPRESSION_LIB);
```

`config.constants.ts`
```
const DEFAULT_COMPRESSION_LIB = 'deflate' 
```

Reply:
https://github.com/dream-sports-labs/delivr-cli/commit/2b6ed541be5fe5660506f09c2d1e0f4f26e5c921